samples/fabric/fabric-cli/scripts/reclaimAsset.sh

```diff
@@ -16,11 +16,10 @@ cd scripts
 # Create a chaincode.json from the template file after replacing user IDs with certificates
```

Comment:
Do we need this script anymore?

Reply:
No, actually Initially I was trying to change with in this, so may be that's why these have come up in this PR.
I can delete the scripts if you are okay with it...

Reply:
Yes, you can delete them. If we ever need to refer to them, we can look at the revision history.